ci/test-conformance-gke.sh

```diff
@@ -240,6 +245,8 @@ function deliver_antrea_to_gke() {
     elif [[ -n ${USER_EMAIL+x} ]]; then
         gcloud projects add-iam-policy-binding ${GKE_PROJECT} --member user:${USER_EMAIL} --role roles/container.admin
         kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user ${USER_EMAIL}
+    elif [[ "$INTEGRATED_CREDENTIAL" == true ]]; then
+        echo "Using the Temporary Cloud Credential for Cluster Management"
```

Comment:
@antoninbas May I know what the clusterrolebinding is used for in gke test? Currently the test can pass without it. For the credential login method, neither the service account nor the email username is provided.

Reply:
I don't know and I didn't write the original version of this script. It's been almost 4 years, so I don't know if @Dyanngg remembers?

At this stage in the script, we have already created the cluster and we have an (admin?) kubeconfig file, so not sure what both of these commands do.

Reply:
I don't remember this specifically, but it must be derived from some documentation at the time when the script is written which could be obsolete at this point

Reply:
@antoninbas I reviewed the script on the credential node and noticed that the user type is PowerUser, indicating that the credential we applied already has permission to manage the cluster.

Reply:
Ok. I wonder if these steps are still required even for other scenarios. But that's a bit orthogonal to this PR.
I am not a big fan of the flag name though. Maybe it should be `--skip-iam-policy-binding`? Or maybe we should just make this the default behavior when neither `SVC_ACCOUNT_NAME` nor `USER_EMAIL` are set?

Reply:
Updated the flag name to `--skip-iam-policy-binding`, PTAL.

Reply:
Creating resource "clusterroles" in API group "rbac.authorization.k8s.io" requires ["container.clusterRoles.create"] permission. I believe this is the reason why we needed to add the ClusterRoleBinding previously. However, since the new account already has this permission(kube admin) in the GKE IAM setup, we no longer need to create this resource.